core-server/src/main/java/org/orbisgis/orbisserver/control/web/IndexController.java

```diff
@@ -84,23 +106,29 @@
     Template index;
 
     /**
-     * The action method returning the index page. It handles
-     * HTTP GET request on the "/" URL.
-     *
-     * @return the index page
+     * The action method returning the html index page containing a list of all the OrbisWPS processes
+     * readable by a human. It handles HTTP GET request on the "/index" URL.
+     * Display a message instead of the list if the method fails.
      */
-    @Route(method = HttpMethod.GET, uri = "/")
+    @Route(method = HttpMethod.GET, uri = "/index")
     public Result index() {
         //Simple example of getting information from the WpsServer
-        try { simpleWpsRequest(); } catch (JAXBException ignored) {}
-        return ok(render(index, "index", "Welcome"));
+        try {
+          getXMLFromGetCapabilities();
+        } catch (JAXBException e) {
+          LOGGER.error(I18N.tr("Unable to get the xml file corresponding to the GetCapabilities request. \nCause : {0}.", e.getMessage()));
+          this.processesList = "Unable to get the xml file";
+        }
+        return ok(render(index,"liste", processesList));
     }
 
+
     /**
-     * Dirty method to display in the logger the list of the processes.
+     * Method to get the xml file corresponding to the GetCapabilities request.
      * @throws JAXBException JAXB Exception.
      */
-    private void simpleWpsRequest() throws JAXBException {
+    public void getXMLFromGetCapabilities() throws JAXBException {
```

Comment:
Maybe this method should return either the `WpsCapabilitiesType` object or the process list instead of setting variables. (Linked to the comment about local variable)